docker-compose.yml

```diff
@@ -105,19 +105,22 @@ services:
                     - mongo-inventory
                     - mongo-useradm
 
-    mender-conductor:
```

Comment:
As this PR is not really modifying mender-conductor, it would be nicer without this re-ordering of the services (so that we don't disturb Git history).

Reply:
> As this PR is not really modifying mender-conductor, it would be nicer without this re-ordering of the services (so that we don't disturb Git history).

this is part of the WIP, I will clean it before review. also: I think we need better health checks, like that:
```yaml
healthcheck:
  test: curl -sS http://mender-elasticsearch:9200 || exit 1
  interval: 5s
  timeout: 4s
  retries: 8
```

Reply:
since the health of all services is defined by the ability to answer to GET on certain endpoint. this should fix more startup issues.

Reply:
redis and mongo can use `nc -z -v ip port`